zerver/tests/test_validators.py

```diff
@@ -31,12 +30,6 @@
     to_wild_value,
 )
 
-if settings.ZILENCER_ENABLED:
-    pass
-
-if TYPE_CHECKING:
-    pass
```

Comment:
Not sure why this was even a thing, unless I'm missing something this seems obviously delete-able.

Reply:
Yeah I think it is; likely just a forgotten bit of code from c9a299a8f856f5e7a4f79f6953783fe9a36cb48a (@PIG208 can confirm).